scripts/launch/connect-to-cluster/default.nix

```diff
@@ -18,6 +18,7 @@
 , disableClientAuth ? false
 , extraParams ? ""
 , iohkPkgs
+, gitrev # workarround for a minor bug
```

Comment:
Could you elaborate on that "minor bug"? I'd just like to understand what I missed.

Reply:
my new `mapOnTest'` function passes gitrev to everything, including the new connect scripts